pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -251,6 +251,10 @@ def _export_video(self, filename: str, res_index: int, profile_index: int, sampl
         if scene_data is None:
             return
 
+        filename = QUrl(filename).path()  # handle the file:// automatically added by Qt/QML
+        if platform.system() == "Windows" and filename.startswith("/"):
+            filename = filename[1:]
```

Comment:
Should we add a function for this? We already have 3 instances of it.

And there is also this one which I assume is also broken:
```python
            elif data["type"] == "file" and val is not None:
                val = QUrl(val).path()  # handle the file:// automatically added by Qt/QML
```

Reply:
Done.